src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -81,6 +81,14 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    endSaleLoader: gravityLoader(
+      id => `sale/${id}/end_sale`,
+      {},
+      {
+        headers: true,
```

Comment:
This option changes the shape of the data which this `Promise` will resolve with.

If either resolves `{body: .., headers: ...}`, or just `{}` (body directly). _Usually_ the reason for wanting to resolve the headers as well is b/c we sometimes include counts in the header `X-Total-Count`.

However, in this case you don't need that, so I would just drop it.